File: ployglots/mta-ployglot-pkg-app-yaml-loader/examples/sdk/_run/02-explicit.py
import logging

from app_yaml_loader import load_files

logging.basicConfig(level=logging.WARNING)

m = load_files(
    [
        "./examples/fixtures/explicit/a.yml",
        "./examples/fixtures/explicit/b.yml",
        "./examples/fixtures/explicit/c.yml",
    ],
    missing="skip",
)
print("keys:", [p.rsplit("/", 1)[-1] for p in m.keys()])

File: ployglots/mta-ployglot-pkg-app-yaml-loader/examples/sdk/_run/02-explicit.mjs
import { loadFiles } from '@ployglot/app-yaml-loader';

const map = await loadFiles(
  ['./examples/fixtures/explicit/a.yml',
   './examples/fixtures/explicit/b.yml',
   './examples/fixtures/explicit/c.yml'],
  { missing: 'skip', logger: console },
);

console.log('keys:', Array.from(map.keys()).map((p) => p.split('/').pop()));
